spacetemplate/assets/legacy.yaml

```diff
@@ -81,7 +96,10 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Papercuts
   can_construct: yes
-  description: TBD
+  description: A papercut is a minor issue, often too small to be individually
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed